Sources/Core/StaticString.swift

```diff
@@ -0,0 +1,74 @@
+// Copyright © 2022 Max Desiatov <max@desiatov.com>.
+// All Rights Reserved.
+// SPDX-License-Identifier: BSD-3
+
+@frozen
+public struct StaticString {
+  @usableFromInline
+  internal var _startPtrOrData: Builtin.Word
```

Comment:
Hmm, not sure I understand the reason for the name here.  Does `StaticString` indicate a small string optimization?

Reply:
Yes, according to API docs, it either stores a pointer to string data, or a single ASCII character.